spec/user_spec.cr

```diff
@@ -190,6 +190,15 @@ module PlaceOS::Model
         found_ids.should eq expected_users.compact_map(&.id).sort!
         found_ids.should_not contain(not_expected.id)
       end
+
+      it "#find_by_email" do
+        existing = Authority.find_by_domain("localhost")
+        authority = existing || Generator.authority.save!
+        expected_user = Generator.user(authority).save!
+
+        found = User.find_by_email(authority.id.as(String), expected_user.email.upcase)
+        found.try(&.id).should eq expected_user.id
+      end
```

Comment:
Could be a describe block with specs for different cases. ATM we're only testing one case.